manifests/argo-rollouts/kind-cluster.yaml

```diff
@@ -0,0 +1,17 @@
+kind: Cluster
+apiVersion: kind.x-k8s.io/v1alpha4
+nodes:
+- role: control-plane
+  kubeadmConfigPatches:
+  - |
+    kind: InitConfiguration
+    nodeRegistration:
+      kubeletExtraArgs:
+        node-labels: "ingress-ready=true"
```

Comment:
Do we need this? Can we comment why this is needed? 🤗

Reply:
I can provide the comment, it is just if people want to try it in a local kind cluster but then that couldn't also be simplified; I guess we can remove the file altogether